draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/FieldMessage.spec.tsx

```diff
@@ -51,5 +51,13 @@ describe("<FieldMessage />", () => {
     const { container } = renderFieldMessage({ status: "error" })
 
     expect(container.querySelector(".warningIcon")).toBeTruthy()
+    expect(screen.getByTitle("Error message"))
+  })
+
+  it("should render a warning icon with an error status", () => {
```

Comment:
Just nitpicking, but I against `shoulds` ;) It "does it" or "does not do it".
```suggestion
  it("renders a warning icon with an error status", () => {
```

I think as a team we should (ha!) make a decision here.